src/NovelRT/Audio/AudioService.cpp

```diff
@@ -79,7 +80,7 @@ ALuint AudioService::readFile(std::string input) {
 std::vector<ALuint>::iterator AudioService::loadMusic(std::string input) {
   if (!isInitialised) {
     _logger.logError("Cannot load new audio into memory while the service is uninitialised! Aborting...");
-    throw std::runtime_error("Unable to continue! Dangerous call being made to AudioService::load. You cannot load new audio when the service is not initialised.");
+    throw NovelRT::Exceptions::NotInitialisedException("AudioService::load", "You cannot load new audio when the service is not initialised.");
```

Comment:
We should standardise/unify the exception message so it is only the function name that is being passed in probably. Something like "Service attempted an operation it cannot perform without being initialised. Method name: " then we just append the "AudioService::Load" string to it probably?

This should make for nicer API UX in all the places we are using this exception.